src/js/constants/deviceConstants.js

```diff
@@ -109,6 +109,35 @@ module.exports = {
       help: `The "regular expression" operator matches the selected field's value with a Perl compatible regular expression (PCRE), automatically anchored by ^. If the regular expression is not valid, the filter will produce no results. If you need to specify options and flags, you can provide the full regex in the format of /regex/flags, for example.`
     }
   },
+  DEVICE_ISSUE_OPTIONS: {
+    offline: {
+      key: 'offline',
+      needsReporting: true,
+      filterRule: {
+        scope: 'system',
+        key: 'updated_ts',
+        operator: '$lt',
+        value: () => {
+          const yesterday = new Date();
+          yesterday.setDate(yesterday.getDate() - 1);
+          return yesterday.toISOString();
+        }
+      },
+      title: 'Offline devices'
+    },
+    monitoring: {
+      key: 'monitoring',
+      needsReporting: false,
+      filterRule: { scope: 'monitoring', key: 'alerts', operator: '$gt', value: 0 },
+      title: 'Devices reporting monitoring issues'
+    },
+    authRequests: {
+      key: 'authRequests',
+      needsReporting: true,
+      filterRule: { scope: 'monitoring', key: 'auth_requests', operator: '$gt', value: 1 },
+      title: 'Devices with new authentication requests'
+    }
+  },
```

Comment:
This is not implemented (as of today). In any case, it wouldn't be `monitoring` scope.

Reply:
In order to provide the functionality as it is defined in the design we should get it implemented though... But once we get the reporting service in we can likely adjust a few places anyway (e.g. finally getting #992 in)...

Reply:
@mzedel does this block the release? we don't have ANY plan today to implement this feature in the backend before the introduction of the reporting service.

Reply:
nono, that doesn't block at all - this (and the offline one) are just in preparation (to check the general approach) and are filtered based on the `needsReporting: true` 👌